common/flatpak-instance-private.h

```diff
@@ -30,8 +30,22 @@ char *flatpak_instance_get_instances_directory (void);
 char *flatpak_instance_allocate_id (char **host_dir_out,
                                     int *lock_fd_out);
 
+gboolean         flatpak_instance_claim_per_app_temp_directory (const char *app_id,
```

Comment:
Really minor thing: there seems to be 8 extra spaces that were not meant to be here.

Reply:
I think in an earlier version of this branch, this function was aligned with another function with a longer return type. I'll drop the extra whitespace in the next revision.